### PR TITLE
Update workflows

### PR DESCRIPTION
**Pull request by IanDelMar**
and get rid of the warning about the deprecated Node.js version.

image

### CODE REVIEW

File: .github/workflows/spelling.yml [hunk 1]
```diff
@@ -9,11 +9,11 @@ on:  # yamllint disable-line rule:truthy
 jobs:
     typos:
         name: "Typos"
-        runs-on: "ubuntu-22.04"
+        runs-on: "ubuntu-latest"
```
[the remark this answers is not in the document]
Comment (IanDelMar) [reply .github/workflows/spelling.yml]: Done. Sorry to waste your time.

[the remark this answers is not in the document]
Comment (szepeviktor) [reply .github/workflows/spelling.yml]: You are very kind to contribute.

[the remark this answers is not in the document]
Comment (szepeviktor) [reply .github/workflows/spelling.yml]: I also like to changing bytes in unused repos.

[the remark this answers is not in the document]
Comment (IanDelMar) [reply .github/workflows/spelling.yml]: It is used by https://github.com/php-stubs/wordpress-stubs and https://github.com/szepeviktor/phpstan-wordpress.

[the remark this answers is not in the document]
Comment (szepeviktor) [reply .github/workflows/spelling.yml]: Yes you are right.
Me is not used by WordPress projects.

[the remark this answers is not in the document]
Comment (szepeviktor) [reply .github/workflows/spelling.yml]: I am sorry that I always talk about my unemployment.
I feel this special kind of pain when I see "WordPress" on my screen.

[the remark this answers is not in the document]
Comment (IanDelMar) [reply .github/workflows/spelling.yml]: No worries, that's absolutely fine in my view. It’s completely understandable to feel that way when things are tough on the work front. I hope you’ll keep engaging with these WP projects. Maybe that leads you to connections and opportunities you don’t expect. Also, your experiences and insights are valuable to others.

**Pull request merged**